src/pages/tutorial/tutorial.js

```diff
@@ -103,6 +104,14 @@ class TutorialPage extends React.Component {
           <Page className="pf-u-h-100vh">
             <SkipToContent href="#main-content">Skip to content</SkipToContent>
             <RoutedConnectedMasthead />
+            <PageSection variant="light">
+              <Breadcrumb
+                threadName={parsedThread.title}
+                isAllSolutionPattern="true"
```

Comment:
Also, check with Tiffany but the breadcrumb should say Solution Patterns aka plural because it will be navigating to the list of all solution patterns, not the main page of this particular solution pattern.

Reply:
Yes, it should be plural. "Solution Patterns"